README.md

```diff
@@ -34,6 +34,6 @@ Any improvements, bug fixes or amendments should also be submitted back as pull
 
 ## Credits
 This plugin was written by Tony Blacker, NHS Leadership Academy Digital Delivery Team. It is based on the tutorial 
-provided by [Morten Rand-Hendriksen](https://mor10.com/) and extended out to match the [NHSUK Frontend library 
-components](https://nhsuk.github.io/nhsuk-frontend/components) 
+provided by [Morten Rand-Hendriksen](https://mor10.com/) and extended out to match the [NHSUK Front-end library 
+components](https://nhsuk.github.io/nhsuk-front-end/components) 
```

Comment:
this will break the link. frontend in this context is a url so please remove hyphenation

Reply:
Done, thanks.